jest.config.js

```diff
@@ -6,5 +6,5 @@ module.exports = {
   transform: {
     "^.+\\.ts$": "ts-jest",
   },
-  setupFilesAfterEnv: ["<rootDir>/lib/index.js"],
+  setupFilesAfterEnv: ["<rootDir>/src/index.ts"],
```

Comment:
@mxschmitt idk if it make much sense, but as i know jest got ability to handle `ts` files only in > 27 versions. So it won't work in previous jest versions

Reply:
Since this is only for our test-suite I think its fine.